pkgs/development/python-modules/pyzstd/default.nix

```diff
@@ -33,6 +35,14 @@ buildPythonPackage rec {
     setuptools
   ];
 
+  dependencies = [
```

Comment:
```suggestion
  dependencies = lib.optionals (pythonOlder "3.13") [
```

Reply:
Noted! This builds without `typing-extensions` from Python 3.13 onward, so technically you are right that this is needed.

Reply:
Which you could learn by reading https://github.com/Rogdham/pyzstd/blob/0.17.0/setup.py#L207